src/kOS/Screen/TermWindow.cs

```diff
@@ -633,7 +642,7 @@ public void ProcessOneInputChar(char ch, TelnetSingletonServer whichTelnet, bool
                             if (whichTelnet == null)
                                 Close();
                             else
-                                whichTelnet.DisconnectFromProcessor();
+                                DetachTelnet(whichTelnet);
```

Comment:
There's really no difference here, other than me thinking "well, if I have this public method to do this, I should probably be using it internally too instead of explicitly doing the same code manually."